tests/bluetooth/tester/src/audio/btp_pbp.c

```diff
@@ -0,0 +1,288 @@
+/* btp_pbp.c - Bluetooth PBP Tester */
+
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "btp/btp.h"
+
+#include <zephyr/bluetooth/audio/pbp.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/util.h>
+#define LOG_MODULE_NAME bttester_pbp
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, CONFIG_BTTESTER_LOG_LEVEL);
+
+#define PBP_EXT_ADV_METADATA_LEN_MAX 128
```

Comment:
Why is this set to 128?

Reply:
cap_broadcast_source_adv_setup was using 128 as an arbitrary large size for base_buf, so I decided to go for the same approach.